src/main/src/main/java/org/geoserver/config/util/XStreamPersister.java

```diff
@@ -2683,20 +2681,44 @@ public boolean canConvert(Class aClass) {
         @Override
         protected void doMarshal(
                 Object source, HierarchicalStreamWriter writer, MarshallingContext context) {
-            GrowableInternationalString internationalString = (GrowableInternationalString) source;
-            Set<Locale> locales = internationalString.getLocales();
-            int size = locales.size();
-            for (Locale l : locales) {
-                if (l != null) {
-                    String elementName = l.toLanguageTag();
-                    if (elementName.contains(" ")) elementName = elementName.replaceAll(" ", "__");
-                    writer.startNode(elementName);
-                    writer.setValue(internationalString.toString(l));
-                    writer.endNode();
-                } else if (size > 1) {
-                    writer.startNode(DEFAULT_LOCALE);
-                    writer.setValue(internationalString.toString(l));
-                    writer.endNode();
+            boolean isGrowable = source instanceof GrowableInternationalString;
+            String defaultValue = "";
+            if (isGrowable) {
+                GrowableInternationalString internationalString =
+                        (GrowableInternationalString) source;
+                Set<Locale> locales = internationalString.getLocales();
+                int size = locales.size();
+                for (Locale l : locales) {
+                    if (l != null) {
+                        String elementName = l.toLanguageTag();
+                        if (elementName.contains(" "))
+                            elementName = elementName.replaceAll(" ", "__");
+                        writer.startNode(elementName);
+                        writer.setValue(internationalString.toString(l));
+                        writer.endNode();
+                    } else if (size > 1) {
+                        writer.startNode(DEFAULT_LOCALE);
+                        writer.setValue(internationalString.toString(l));
+                        writer.endNode();
+                    }
+                }
+            } else {
+                defaultValue = source.toString();
+                Set<Locale> locales = Stream.of(Locale.getDefault()).collect(Collectors.toSet());
+                int size = locales.size();
+                for (Locale l : locales) {
+                    if (l != null) {
+                        String elementName = l.toLanguageTag();
+                        if (elementName.contains(" "))
+                            elementName = elementName.replaceAll(" ", "__");
+                        writer.startNode(elementName);
+                        writer.setValue(defaultValue);
+                        writer.endNode();
+                    } else if (size > 1) {
+                        writer.startNode(DEFAULT_LOCALE);
+                        writer.setValue(defaultValue);
+                        writer.endNode();
+                    }
```

Comment:
this code block is the same used for the GrowableInternationalString case. It would be better to put it into a separate method to avoid duplication